frontend/src/config/cambodia/layers.json

```diff
@@ -1943,5 +1943,21 @@
         "color": "#b03060"
       }
     ]
+  },
+  "cdi": {
+    "title": "Combined Drought Index",
+    "type": "composite",
+    "base_url": "https://to-fill-later",
```

Comment:
We probably don't need the base_url since we will use the urls of the underlying layers